includes/wf_crm_webform_postprocess.inc

```diff
@@ -2206,4 +2207,14 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
 
     return $reorderedArray;
   }
+
+  private function unsetEmptyValueIndexes($values, $entity) {
+    foreach($values as $k => $v) {
+      if(!isset($v[$entity])) {
```

Comment:
Fyi Drupal coding standards are for a space to follow a control statement (so a space after `foreach` and `if` before the opening parenthesis.